Sources/AblyAssetTrackingPublisher/Services/DefaultLocationService.swift

```diff
@@ -71,6 +71,16 @@ class DefaultLocationService: LocationService {
         locationManager.systemLocationManager.stopUpdatingLocation()
     }
     
+    func requestLocationUpdate() {
+        /*
+         From reading the ``CLLocationManager/startUpdatingLocation`` documentation, this seems to be the only programmatic way to provoke it into emitting a location event:
+        
+        > Calling this method several times in succession does not automatically result in new events being generated. Calling stopUpdatingLocation() in between, however, does cause a new initial event to be sent the next time you call this method.
+        */
+        locationManager.systemLocationManager.stopUpdatingLocation()
+        locationManager.systemLocationManager.startUpdatingLocation()
```

Comment:
I wonder how this affects the Mapbox SDK 🤔 
1. Have you checked whether this doesn't break things like location history recording? 
2. Will there be a big delay in location updates between the "stop" and "start" operations?
3. Additionally, do we know what happens when you stop and start location updates? Will it consume a lot of resources to do that?

Reply:
These are all good questions, I'll look into it and get back to you 🙂

Reply:
I've added [a script](https://github.com/ably/ably-asset-tracking-swift/tree/425-only-first-trackable-added-comes-online-when-stationary/Tools/Analysis/AnalyzeLocationEventFrequencies) for analysing the logs from the publisher example app, to find out if the frequencies of location updates and recorded locations is affected. My brief investigation using the iOS simulator suggests that they are not. However, I want to test on a device too, and for that we'll probably need to add the ability to log to a file. Once I've done that and gathered some results from a device, I'll post my findings here in detail.

As for resources (which I'm mainly taking to mean battery life), I do not know what the impact would be. I could have a think about how to investigate it, but I don't have any ideas off the top of my head.